src/types.ts

```diff
@@ -18,7 +18,51 @@ export const DEFAULT_SEARCH_PARAMS = {
     propertyGroups: ['All'],
     managed: ['All'],
     readyForSearch: false,
+    landArea: {
+        min: 0,
```

Comment:
I've changed some stuff in types in the sales evidence filter branch. We'll merge this into master and then merge my branch into master and I'll fix the conflicts then.